projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4020,12 +4019,12 @@ public function admin_page_load() {
 					wp_safe_redirect( self::admin_url( 'page=jetpack' ) );
 					exit;
 				case 'unlink':
-					$redirect = isset( $_GET['redirect'] ) ? $_GET['redirect'] : '';
+					$redirect = isset( $_GET['redirect'] ) ? $_GET['redirect'] : ''; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
Shouldn't be ignored.

Reply:
d83ff62466ddbfca1f204356944a763e95508f4d